Vish/assignment-5/VishnuA5_1.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Document</title>
+    </head>
+    <body>
+        <header class="headerBox">
+            <h1>Vishnu Maddheshiya</h1>
+        </header>
+        <section class="hero">
+
+            <h3 class="left-content">About Me</h3>
+            <div class="about me">
+                <ol>
+                    <li>My name is Vishnu Maddheshiya.</li>
+                    <li>I am from Uttar Pradesh (Mau) 275101.</li>
+                    <li>Talking about my educational qualification, I have completed my BCA from IICS College.</li>
+                    <li>I have done my 12th in DAV Inter college and 10th in our DAV Inter College.</li>
+                    <li>Talking about my family, my family consists of 7 members including me and my elder brother. </li>
+                    <li>My father is a farmer and my mother is a housewife.</li>
+                    <li>My hobbies are cooking, listening to music and playing the shuttle.</li>
+                </ol>
+            </div>
+            <h3 class="left-content mau">MoreAbout Me</h3>
+            <div class="about MoreAbout">
+                <ul type=disk>
+                    <li><a href="https://www.facebook.com/profile.php?id=100008154193219" target="_blank"> My Resume </a></li>
```

Comment:
change the shape of bullets colour them and apply border.

Reply:
kya sanket main samjha nahi

Reply:
thume ul ke bullets ko square shape dena hai aur usko color aur border dena hai

Reply:
i change it

Reply:
maine CSS se add kar diya hai use